### ISSUE TITLE
[SF]Card: [A11y AXE Error]Landmarks should have a unique role or role/label/title (i.e. accessible name) combination

### ISSUE THREAD

**Issue by VidhiNPatel** [SAP/ui5-webcomponents-react#7337]
### Describe the bug

Using Axe Devtools, the Card  has identified an a11y issue with moderate impact: `Landmarks should have a unique role or role/label/title (i.e. accessible name) combination`

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/data-display-card--docs

### Reproduction steps

1. Go to the storybook for https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/data-display-card--docs
2. Open Axe devtools in dev console
3. Issue is highlighted for Card component

### Expected Behaviour

https://dequeuniversity.com/rules/axe/4.10/landmark-unique?application=AxeChrome

### UI5 Web Components for React Version

2.10

### UI5 Web Components Version

2.9

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @VidhiNPatel 

please make sure that this is really an issue with the component and not only with the "Docs" page. If you identified it as component problem please add at least a clear description of the problem including screenshots and the version of the tool which identified the issue.

**Comment by VidhiNPatel** [comment]
By adding the accessibleName property on the card, issue will be fixed. Hence closing it
